src/main/java/com/corndel/pixmate/drawings/Checkerboard.java

```diff
@@ -0,0 +1,25 @@
+package com.corndel.pixmate.drawings;
+
+public class Checkerboard {
+
+    public static void main(String[] args) {
+
+
+                int rows = 5;  // Number of rows to print
+
+                for (int i = 0; i < rows; i++) {
```

Comment:
Nice work around, 100% simpler then using a nested loop. If you get a chance have a go and use a nested loop 😉